editor/gui/inspector_dock.cpp

```diff
@@ -147,6 +148,10 @@ void InspectorDock::draw(float deltaMilliseconds)
 
 				ImGui::Separator();
 
+				m_OpenedEntity->getScript()->draw();
```

Comment:
What happens if `getScript()` returns nullptr?

Reply:
will never happen 😉

Reply:
Now it can

Reply:
N O